services/skus/service_nonint_test.go

```diff
@@ -4656,8 +4656,208 @@ func Test_newRadomGateway(t *testing.T) {
 	}
 }
 
+func TestService_processRadomEvent(t *testing.T) {
+	type tcGiven struct {
+		event *radom.Event
+	}
+
+	type tcExpected struct {
+		err error
+	}
+
+	type testCase struct {
+		name  string
+		given tcGiven
+		exp   tcExpected
+	}
+
+	tests := []testCase{
+		{
+			name: "should_not_process",
+			given: tcGiven{
+				event: &radom.Event{},
+			},
+		},
+	}
+
+	for i := range tests {
+		tc := tests[i]
+
+		t.Run(tc.name, func(t *testing.T) {
+			s := Service{}
+
+			actual := s.processRadomEvent(context.Background(), tc.given.event)
+
+			should.ErrorIs(t, actual, tc.exp.err)
+		})
+	}
+}
+
+func TestService_processRadomEventTx(t *testing.T) {
```

Comment:
suggestion: At some point, it would be nice to see more test cases covering some of the important branches of the method. Currently it covers only the happy path, which does not guarantee the expected behaviour under error conditions.

nit: Above and below, the variable for service should hold a pointer as that's what Service usually is, and that's how it's declared in other places. I know that Go will properly handle this when calling the method, however, visually, when reading the code, the inconsistency is noticeable and distracting – there is this urge to fix it.